bin/do_Pk1D.py

```diff
@@ -126,7 +126,7 @@ def compute_mean_delta(ll,delta,iv,zqso):
 #   Create root file
     if (args.out_format=='root') :
         from ROOT import TCanvas, TH1D, TFile, TTree, TProfile2D, TProfile
-        storeFile = TFile("Testpicca.root","RECREATE","PK 1D studies studies");
+        storeFile = TFile(args.out_dir+"Testpicca.root","RECREATE","PK 1D studies studies");
```

Comment:
It's safer to add a `/` in case the user doesn't include it:

    storeFile = TFile(args.out_dir+"/Testpicca.root"...